modules/bvl_feedback/ajax/react_get_bvl_threads.php

```diff
@@ -2,46 +2,48 @@
 /**
  * Used by react on initial load to load the thread state.
  *
- * PHP version 5
+ * PHP version 7
  *
  * @category Behavioural
  * @package  Main
  * @author   Evan McIlroy <evanmcilroy@gmail.com>
  * @license  GPLv3 <http://www.gnu.org/licenses/gpl-3.0.en.html>
  * @link     https://www.github.com/aces/Loris-Trunk/
  */
+namespace LORIS\bvl_feedback;
+use \LORIS\StudyEntities\Candidate\CandID;
 header("content-type:application/json");
 ini_set('default_charset', 'utf-8');
 
 set_include_path(
-    __DIR__ . "/../../../project/libraries:" .
```

Comment:
`__DIR__` doesn't have a trailing slash (https://secure.php.net/manual/en/language.constants.predefined.php), so this is wrong